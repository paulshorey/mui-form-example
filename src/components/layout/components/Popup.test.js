import React from 'react';
import renderer from 'react-test-renderer';

// shallow + redux
import { shallow, connected, shallowConnected } from 'index.testing';
import { Popup } from './Popup';

it('renders correctly', () => {
	const tree = renderer.create(shallowConnected(<Popup />)).toJSON();
	expect(tree).toMatchSnapshot();
});
