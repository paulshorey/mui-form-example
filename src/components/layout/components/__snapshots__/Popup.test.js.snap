// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="popupBox  sc-bdVaJa cnOeLs"
>
  <div
    className="popupBox_overlay"
    onClick={[Function]}
  />
  <div
    className="popupBox_content"
  >
    <div
      className="popupBox_close"
      onClick={[Function]}
    >
      <span
        className="fontIcon icon-cross"
        title="Close"
      />
    </div>
  </div>
</div>
`;
